Makefile

```diff
@@ -4,4 +4,6 @@ GO=CGO_ENABLED=1 GO111MODULE=on go
 
 test:
 	$(GO) test ./... -cover
-	$(GO) vet ./...
\ No newline at end of file
+	$(GO) vet ./...
+	gofmt -l .
+	[ "`gofmt -l -w .`" = "" ]
```

Comment:
actually I don't think this should change the files, so drop `-w`. See https://github.com/edgexfoundry/go-mod-registry/pull/8#discussion_r271117719